results/24-RF-selected-LV-TableStats.Rmd

```diff
@@ -129,7 +134,7 @@ We already calculated the metaviper correlations here
 
 ##get correlated metaviper scores
 metaviper_scores='syn21259610'
-mtab<-synapser::synTableQuery(paste('SELECT * FROM syn21322923 where corVal>0.65'))$asDataFrame()%>%
+mtab<-synapser::synTableQuery(paste('SELECT * FROM syn21322923 where corVal>0.75'))$asDataFrame()%>%
```

Comment:
not sure why the corVal is changed to 0.75

Reply:
Because lower plotted too many points.